lib/aur-build

```diff
@@ -106,7 +107,7 @@ if (($#)); then
     fi
 fi
 
-if type -P xdelta3 >/dev/null; then
+if type -P xdelta3 >/dev/null && ((delta)); then
```

Comment:
If there's a flag, we can probably leave out the `type -P`

Reply:
dropped `type -P` conditional
added disclaimer to `aur-build` man page pointing to `xdelta3`